polliwog/polyline/_try_inflection_points.py

```diff
@@ -53,7 +53,7 @@ def main():
             )
             result_points.append(result)
         except ValueError:
-            pass
+            print("no point of max acceleration found.")
```

Comment:
The behavior is fine as it is – there is no need to print anything. I think what the tooling wants is a comment to explain the `pass`.

```suggestion
            # No-op if this point doesn't exist.
            pass
```